exampleCourse/courseInstances/Sp15/assessments/hw01-templateQuestions/infoAssessment.json

```
@@ -0,0 +1,24 @@
+{
+    "uuid": "597176b9-9145-4226-8a77-d0973bad80df",
+    "type": "Homework",
+    "title": "Template homework - copy this to start a new HW",
+    "set": "Homework",
+    "number": "1",
+    "allowAccess": [
+        {
+            "role": "Instructor",
+            "credit": 100
+        }
+    ],
+    "zones": [
+        {
+            "questions": [
+                {"id": "templateCheckbox", "points": 1, "maxPoints": 2},
+                {"id": "templateFixedMultipleChoiceSingleAnswer", "points": 1},
+                {"id": "templateFixedMultipleChoiceManyAnswers", "points": 1, "maxPoints": 2},
+                {"id": "templateRandomMultipleChoiceListPrompt", "points": 1, "maxPoints": 4},
+                {"id": "templateRandomMultipleChoiceNumericChoices", "points": 1, "maxPoints": 4}
```

```suggestion
                {"id": "templateMultipleChoiceSingleCorrect", "points": 1},
                {"id": "templateMultipleChoiceMultipleCorrect", "points": 1, "maxPoints": 2},
                {"id": "templateMultipleChoiceFromList", "points": 1, "maxPoints": 4},
                {"id": "templateMultipleChoiceNumeric", "points": 1, "maxPoints": 4}
```

I know that Fixed/Random was being used in the sense of "does the Python code do randomization", but I was worried it was misleading because even the Fixed variants are actually randomizing, just in the element itself.

---

Updated names in new branch